src/components/NotificationFeed/NotificationFeed.tsx

```diff
@@ -23,7 +28,7 @@ export interface NotificationFeedProps {
   renderItem?: RenderItem;
   onNotificationClick?: OnNotificationClick;
   onMarkAllAsReadClick?: (e: React.MouseEvent, unreadItems: FeedItem[]) => void;
-  isVisible: boolean;
+  isVisible?: boolean;
```

Comment:
`NotificationFeed` doesn't seem to use `isVisible` anymore, should it be removed from `NotificationFeedProps`?

Reply:
Yeah we can – my only hesitation was that we'd cause typescript errors for folks on upgrade but probably not a bad thing?